generators/client/templates/react/package.json.ejs

```diff
@@ -166,6 +166,7 @@ limitations under the License.
     <%_ if (protractorTests) { _%>
     "webdriver-manager": "12.1.4",
     <%_ } _%>
+    "@openapitools/openapi-generator-cli": "0.0.9-3.3.4",
```

Comment:
Better to put 0.0.9-4.0.2

See https://github.com/jhipster/generator-jhipster/pull/9925

Reply:
updated to 0.0.14-4.0.2